### PR TITLE
Create avinashji1302.json

### PR DESCRIPTION
**Pull request by avinashji1302**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7654"></a>

### CODE REVIEW

File: avinashji1302.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      "name": "Follow me on Github",
+      "url": "https://github.com/avinashji1302",
+      "icon": "FaGithub"
+    },
+    
+     {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/avinash-gupta-ab24231b6/",
+      "icon": "FaLinkedin"
+    },
```
Comment (jaydeepkhatri) [inline avinashji1302.json:15]:
```suggestion
    }
```

Remove the `,` after the `}` It makes the JSON invalid

**Review by jaydeepkhatri — CHANGES_REQUESTED** [review]
Hi @avinashji1302 
Thank you for contributing to Linkfree. 

1. I have noticed the JSON is invalid. You can always test your JSON code here -> https://linkfree.io/playground
2. The file `avinashji1302.json` needs to be placed inside the `/data` folder. [FYI](https://github.com/EddieHubCommunity/LinkFree/tree/main/data)

File: avinashji1302.json (hunk 1 above)
Comment (avinashji1302) [reply avinashji1302.json]: > Remove the `,` after the `}` It makes the JSON invalid

Thank You , I am making it correct

Comment (avinashji1302) [reply avinashji1302.json]: > Remove the `,` after the `}` It makes the JSON invalid

I did it , Please check

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
The path has a space in it

```diff
- data /avinashji1302.json
+ data/avinashji1302.json
```

**Review by avinashji1302 — COMMENTED** [review]
Please check !

**Review by avinashji1302 — COMMENTED** [review]
Please check now!

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**